csaf_2.1/comment_resolution/log_from_csd01pr.md

```diff
@@ -16,47 +16,47 @@ The following abbreviations are used:
-| [16](https://groups.oasis-open.org/discussion/csdpr01-feedback-1059-from-tc-member) | 2025-06-16 | JM | [Inconsistencies with use of numerals vs words](https://github.com/oasis-tcs/csaf/issues/1059) |  |  |  |
+| [16](https://groups.oasis-open.org/discussion/csdpr01-feedback-1059-from-tc-member) | 2025-06-16 | JM | [Inconsistencies with use of numerals vs words](https://github.com/oasis-tcs/csaf/issues/1059) | 2025-08-27 | [Comment Resolution CSD011PR: #1059](https://github.com/oasis-tcs/csaf/pull/1153)] | Completed |
 | [17](https://groups.oasis-open.org/discussion/potential-error-in-test-file) | 2025-06-17 | Dominik Burgdörfer | [Potential error in test file](https://github.com/oasis-tcs/csaf/issues/1060) | 2025-08-20 | [Comment Resolution CSD01PR: #1060](https://github.com/oasis-tcs/csaf/pull/1132) | Completed |
-| [18](https://groups.oasis-open.org/discussion/csdpr01-feedback-1062-from-tc-member) | 2025-06-17 | JM | [Appendix A. Acknowledgements TC Members lists need updating](https://github.com/oasis-tcs/csaf/issues/1062) |  |  |  |
+| [18](https://groups.oasis-open.org/discussion/csdpr01-feedback-1062-from-tc-member) | 2025-06-17 | JM | [Appendix A. Acknowledgements TC Members lists need updating](https://github.com/oasis-tcs/csaf/issues/1062) | 2025-08-27 | [Comment Resolution CSD01PR: #1062](https://github.com/oasis-tcs/csaf/pull/1155)] | Completed |
 | [19](https://groups.oasis-open.org/discussion/csdpr01-feedback-1063-from-tc-member) | 2025-06-17 | JM | [3.2.2.5 Document Property - Distribution - Sharing Group changes](https://github.com/oasis-tcs/csaf/issues/1063) | 2025-07-21 | [Sharing group correction and Distribution improvement](https://github.com/oasis-tcs/csaf/pull/1098) | Completed |
 | [20](https://groups.oasis-open.org/discussion/csdpr01-feedback-1061-from-tc-member) | 2025-06-17 | TS | [Informative comments not rendered correctly](https://github.com/oasis-tcs/csaf/issues/1061) | 2025-07-21 | [Fix informative comment rendering](https://github.com/oasis-tcs/csaf/pull/1099) | Completed |
 | [21](https://groups.oasis-open.org/discussion/csdpr01-feedback-1064-from-tc-member) | 2025-06-17 | TS | [Invalid references](https://github.com/oasis-tcs/csaf/issues/1064) | 2025-07-18 | [Fix invalid references](https://github.com/oasis-tcs/csaf/pull/1082) | Completed |
-| [22](https://groups.oasis-open.org/discussion/csdpr01-feedback-1065-from-tc-member) | 2025-06-17 | TS | [`sec` instead of section number](https://github.com/oasis-tcs/csaf/issues/1065) |  |  |  |
+| [22](https://groups.oasis-open.org/discussion/csdpr01-feedback-1065-from-tc-member) | 2025-06-17 | TS | [`sec` instead of section number](https://github.com/oasis-tcs/csaf/issues/1065) | 2025-08-27 | [`sec` instead of section number](https://github.com/oasis-tcs/csaf/issues/1065) | Completed |
```

Comment:
```suggestion
| [22](https://groups.oasis-open.org/discussion/csdpr01-feedback-1065-from-tc-member) | 2025-06-17 | TS | [`sec` instead of section number](https://github.com/oasis-tcs/csaf/issues/1065) | 2025-08-27 | Addressed through tool updates; resolved in [CSAF 2.1 CSD01 Public Feedback Resolution and Editor Revision](https://github.com/oasis-tcs/csaf/pull/1108) | Completed |
```